coderd/database/queries/groups.sql

```diff
@@ -8,6 +8,24 @@ WHERE
 LIMIT
 	1;
 
+-- name: ValidateGroupIDs :one
+WITH input AS (
+	SELECT
+		unnest(@group_ids::uuid[]) AS id
+)
+SELECT
+	array_agg(input.id)::uuid[] as invalid_group_ids,
+	COUNT(*) = 0 as ok
+FROM
+	-- Preserve rows where there is not a matching left (groups) row for each
+	-- right (input) row...
+	users
```

Comment:
Should this be `groups`?
```suggestion
	groups
```

Reply:
STEVEN I'M BLIND, I'VE BEEN TRYING TO FIGURE THIS OUT ALL MORNING